src/models/Category.js

```diff
@@ -0,0 +1,39 @@
+import { DataTypes } from 'sequelize';
+import { sequelize } from '../utils/db.js';
+
+const Category = sequelize.define(
+  'category',
+  {
```

Comment:
you should have `id` as primary key and use `guid`

Reply:
but I have it already 😊 
I was using the default id primary key property given by Sequelize ( integer, autoincrement). Same for other models
<img width="270" alt="Screenshot 2023-08-23 at 13 58 53" src="https://github.com/mate-academy/node_accounting-app-with-db/assets/76661298/0fc0c17a-dd26-457e-8b49-920becb34f3d">